analysis/cQED_dataAnalysis/fitrabi.m

```diff
@@ -30,8 +29,8 @@
 % model A + B Exp(-t/tau) * cos(w t + phi)
 rabif = inline('p(1) + p(2)*exp(-tdata/p(3)).*cos(p(4)*tdata + p(5))','p','tdata');
 
-% initial guess for amplitude is y(1) - mean
-amp = y(1) - mean(y);
+% initial guess for amplitude is y(max) - mean
+amp = y(max) - mean(y);
```

Comment:
Sorry about that.  It was one of my repos.  I've pushed it to Qlab as feature/2Dimprovements.  Should I rebase on develop?